frontend/src/features/admin-form/responses/FeedbackPage/FeedbackPage.tsx

```diff
@@ -57,43 +139,87 @@ export const FeedbackPage = (): JSX.Element => {
     >
       <Grid
         mb="1rem"
+        minH="4rem"
         alignItems="end"
         color="secondary.500"
-        gridTemplateColumns={{ base: 'auto', md: 'auto auto 1fr' }}
+        gridTemplateColumns={{ base: 'auto', md: 'auto 7fr 1fr' }}
         gridGap={{ base: '0.5rem', md: '1.5rem' }}
         gridTemplateAreas={{
-          base: "'submissions submissions' 'score export'",
-          md: "'score submissions export'",
+          base: "'information' 'feedbackType' 'export'",
+          md: "'information feedbackType export'",
         }}
       >
-        <Flex gridArea="score" flexDir="column">
-          <Text textStyle="caption-2" color="secondary.400">
-            Average Score
-          </Text>
-          <Text textStyle="display-2">{prettifiedAverageScore}</Text>
-        </Flex>
-        <Box gridArea="submissions">
-          <Text textStyle="h4" mb="0.5rem">
-            <Text as="span" color="primary.500">
-              {count}
-            </Text>
-            {prettifiedFeedbackCount}
-          </Text>
-        </Box>
+        <Container gridArea="information">
+          {getInformationGridComponent(
+            currentFeedbackType,
+            issueProps,
+            reviewProps,
+          )}
```

Comment:
Can be simplified into:
```suggestion
          {currentFeedbackType === FeedbackType.Issues ? getIssueInformationComponent(issueProps.count) : getReviewInformationComponent(reviewProps.average, reviewProps.count)}
```
Then the function definition for `getInformationGridComponent` below can be removed as well.

Reply:
Actually I did this in purpose, it looks too cluttered especially when the texts are long, wdyt?